catalogData/redis32/ha/3x/k8s/sentinel-proxy-deployment.json

```diff
@@ -0,0 +1,45 @@
+{
+  "apiVersion": "apps/v1beta1",
+  "kind": "Deployment",
+  "metadata": {
+    "name": "redis-sentinel-proxy"
+  },
+  "spec": {
+    "replicas": 1,
```

Comment:
Do we need to run more than one of these? What happens to redis during a rolling update when the proxy pod goes down?